src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>net452</TargetFrameworks>
```

Comment:
@gucalder any reason you want your tests to be targeting .NET 452?
If this was a mistake please remove